packages/foreman/rubygem-activerecord-session_store/rubygem-activerecord-session_store.spec

```diff
@@ -74,6 +74,9 @@ cp -a .%{gem_dir}/* \
 %doc %{gem_instdir}/README.md
 
 %changelog
+* Thu Aug 16 2018 Eric D. Helms <ericdhelms@gmail.com> - 1.1.1-2
+- Rebuild for Rails 5.2
```

Comment:
Technically the rebuild is more for the Ruby 2.5 SCL than Rails

Reply:
Technically no :) The Rails 5.2 SCL change (from tfm-ror51 to tfm-ror52) is what necessitates the rebuilding. Whether Ruby changed or not would still result in rebuilding for Rails 5.2. And vice versa. So you could argue both, but primarily Rails is the driver.